datastore/modelassertmanager.go

```diff
@@ -155,41 +154,46 @@ func (db *DB) deleteModelAssert(modelID int) error {
 
 	_, err = db.Exec(deleteModelAssertSQL, modelID)
 	if err != nil {
-		log.Printf("Error deleting the model assertion: %v\n", err)
```

Comment:
Any reason to drop the logging here? Given the raised error is not always caught/displayed later, this maybe the only information we keep in this case (similar for other cases below too).

Reply:
Ah you're right, one of the callers is not handling this error at all.  I fixed that and went through the other funcs below where I've eliminated logging of errors that are getting thrown to make sure that callers were also either logging or throwing errors.